content/shmem_broadcast.tex

```diff
@@ -72,12 +72,15 @@
     needed to ensure this): The \VAR{pSync} array on all \ac{PE}s in the
     \activeset{} is not still in use from a prior call to a broadcast routine.  The
     \dest{} array on all \ac{PE}s in the \activeset{} is ready to accept the
-    broadcast data.
+    broadcast data. \newtext{The data in the \source{} array on the root \ac{PE} is ready to be broadcasted.}
```

Comment:
> Here's another example from the T3E documentation (just to keep picking on Cray): http://docs.cray.com/books/2178_3.0.1/html-2178_3.0.1/z822949834dep.html

@jdinan Even in this example, they already have a barrier_all, and it will be moved to a different place with this clarification. You don't require more than that.